src/actors/symbolication.rs

```diff
@@ -1051,11 +1051,13 @@ impl SymbolicationActor {
                     let cfi_modules = state
                         .referenced_modules()
                         .into_iter()
-                        .filter_map(|code_module| {
-                            Some((
-                                code_module.id()?,
+                        .map(|code_module| {
+                            (
+                                code_module
+                                    .id()
+                                    .unwrap_or_else(|| CodeModuleId::from(DebugId::nil())),
```

Comment:
_nit_: `CodeModuleId` implements `Default`, which could come in handy here:

```suggestion
                                    .unwrap_or_default(),
```